### PR TITLE
Migrate terraform irefindex to VSC

### PR DESCRIPTION
**Pull request by Arthurdw**
Allows the irefindex to be provisioned more autonomously on the VSC.

## Features

- [x] Create an instance on the VSC
- [x] Allow ssh
- [x] Run ansible scripts
- [x] Automatically generate keypair for machine based on local public key
- [x] Create volume for instance

### CODE REVIEW

**Comment by Arthurdw** [comment]
~ We need to create a volume

File: vsc/.gitignore
Comment (MaybeJustJames) [inline vsc/.gitignore:1]: I'd like a `.gitignore` in the top-level directory please.

**Review by MaybeJustJames — APPROVED** [review]
Very well done thank you. I have a small comment which you're welcome to action here or in a later PR. You're welcome to (rebase) merge.

File: vsc/.gitignore
Comment (Arthurdw) [reply vsc/.gitignore:1]: I have moved it. Is there any specific reason not to have directory-level gitignore(s)?

**Pull request merged**